### DISCUSSION TITLE
Build formily antd design

### DISCUSSION THREAD

**Discussion by rezamomeni-dev** [alibaba/designable#215] [General]
### Build Functionality
I want to get a build file from formily>antd project. 
What is the build command line?
I ran "**npm run build:playground**" and it was done perfectly, but when I want to run the build file, It has an error and you can see the error text below.

![error](https://user-images.githubusercontent.com/30400627/152930745-be6aa2a7-e5b2-4dac-a594-36966a1ba812.PNG)

What should I do to get build from formily>antd project? 
Please guide me.
Thanks so much.

**Comment by donkeylucky** [comment]
执行 yarn workspace @designable/formily-antd build:playground 也是同样问题 ！！！

**Reply by donkeylucky** [reply]
解决了 给formily/antd/playground/template.ejs 加一个 &lt;meta charset=&quot;utf-8&quot;&gt;
